src/app/logic/Game.ts

```diff
@@ -0,0 +1,152 @@
+import { HerdConfigInterface } from '../../Interfaces/HerdConfigInterface';
+import { GameModes } from '../../Enums/GameModeEnums';
+import { GameConfigInterface } from '../../Interfaces/GameConfigInterface';
+import { Player } from '../../Player';
+import { BreedProcessor } from '../BreedProcessor';
+import { Dice } from '../Dice';
+import { FirstDice } from '../FirstDice';
+import { SecondDice } from '../SecondDice';
+import { Timer } from '../Timer';
+import { Trade } from '../Trade';
+import { Bank } from './Bank';
+import { defaultGameConfiguration } from './defaultGameConfiguration';
+import { LivestockConfigInterface } from '../../Interfaces/LivestockConfigInterface';
+import { ProtectorsConfigInterface } from '../../Interfaces/ProtectorsConfigInterface';
+import { HerdOwners } from '../../Enums/HerdOwnerEnum';
+
+export class Game {
+  mode: GameModes;
+  roundTimeInSeconds: number;
+  // totalGameTimeInSeconds: number;
```

Comment:
remove comment

Reply:
This a feature that might be introduced in the future